### PR TITLE
update_tutorial environment

### PR DESCRIPTION
**Pull request by tpike3**
## Summary
Improves tutorials by choosing free environments that reduces the cost of entry

## Motive
Colab does not render Solara
Not everyone has a google account

## Implementation
- Adds a Binder link in intro for non-google account holders
- Removes Colab link for visual as Solara does not render properly
- Adds py.cafe link for visualization tutorial
- Adds binder link for visualization tutorial
- 
## Usage Examples
[py.cafe](https://py.cafe/app/maartenbreddels/mesa-example-wealth)

## Additional Notes
Needed ASAP for CSSSA Tutorial

### CODE REVIEW

**Comment by quaquel** [comment]
I have updated the latest branch. It was breaking RTD. Waiting to see if the update fixes it, but I guess you need the docs up an running?

**Comment by tpike3** [comment]
> I have updated the latest branch. It was breaking RTD. Waiting to see if the update fixes it, but I guess you need the docs up an running?

Yep  just fixed I had forgotten to remove some code I messing around with on building a py.cafe badge... this one should work... but give me a sec.. once the build is complete I need to check binder ot make sure it worked as well

**Comment by quaquel** [comment]
I checked the build problem. The tutorial uses `svgwrite `, but this is not installed on RTD (or as part of mesa). Why do you need this dependency?

I am about to head out and won't return for several hours. I'll check in 30 minutes for any response.

**Comment by quaquel** [comment]
I am pretty sure the problem still persists. It is indeed the very last cell in the tutorial that needs to be removed.

**Comment by tpike3** [comment]
> I am pretty sure the problem still persists. It is indeed the very last cell in the tutorial that needs to be removed.

Yep.. I must have fat moused something in my git as I just had all sorts of strange problems.. regardless.. just removed it

**Review by quaquel — APPROVED** [review]

**Comment by quaquel** [comment]
I approved it, so go ahead and merge it.

**Comment by tpike3** [comment]
> I approved it, so go ahead and merge it.

Will do waiting to see if binder builds properly. The build worked from my repo but got make sure I got all the pieces right. However, 1st build always takes forever

File: .virtual_documents/docs/tutorials/intro_tutorial.ipynb
[the remark this answers is not in the document]
Comment (quaquel) [reply .virtual_documents/docs/tutorials/intro_tutorial.ipynb:1]: Yes ideally we don't duplicate. However toon is about to give his mesa tutorial. 

@tpike3  if you merge, can you open an issue about this file duplication?

[the remark this answers is not in the document]
Comment (tpike3) [reply .virtual_documents/docs/tutorials/intro_tutorial.ipynb:1]: Yes and I think this is a .gitignore issue.  I just want to do some background research on their purpose and the specific reason this are autogenerated with readthedocs

[the remark this answers is not in the document]
Comment (EwoutH) [reply .virtual_documents/docs/tutorials/intro_tutorial.ipynb:1]: So to confirm, these files aren’t needed to be tracked in git and can be removed from our repo?

[the remark this answers is not in the document]
Comment (EwoutH) [reply .virtual_documents/docs/tutorials/intro_tutorial.ipynb:1]: So can or can’t these files be removed?

[the remark this answers is not in the document]
Comment (quaquel) [reply .virtual_documents/docs/tutorials/intro_tutorial.ipynb:1]: There is an open issue on it: #2412.

my hunch is that the can be removed and added to gitignore but someone (@tpike3?) should test this.

**Pull request merged**